vhdbuilder/packer/generate-vhd-publishing-info.sh

```diff
@@ -64,17 +77,32 @@ echo "The generated sas token works"
 echo "COPY ME ---> ${STORAGE_ACCT_BLOB_URL}/${VHD_NAME}?***"
 
 # Note: The offer_name is the value from OS_SKU (eg. Ubuntu)
-cat <<EOF > vhd-publishing-info.json
+if [ "${OS_NAME,,}" == "linux" ]; then
+    cat <<EOF > vhd-publishing-info.json
+{
+    "vhd_url": "$vhd_url",
```

Comment:
ummm, are we sure we wont break release scripts with the removed whitespace?

Reply:
seems correct to me:

{
    "vhd_url": "https://vhdbuildereastustest.blob.core.windows.net/vhd/1.1700243552.22224.vhd?st=2023-11-16T00%3A00Z&se=2024-11-17T00%3A00Z&sp=rl&sv=2018-11-09&sr=c&sig=De0NwDmGNeBXpHfZDlHamP92pWD37MTo49RyDJ7NPiI%3D",
    "captured_sig_resource_id": "/subscriptions/8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8/resourceGroups/aksvhdtestbuildrg/providers/Microsoft.Compute/galleries/PackerSigGalleryEastUS/images/2204Gen2/versions/1.1700243552.22224",
    "os_name": "Linux",
    "sku_name": "2204gen2containerd",
    "offer_name": "Ubuntu",
    "hyperv_generation": "V2",
    "image_architecture": "x64",
    "image_version": "0.0.1"
}

also verified this works properly with parsing json for abe2e, would assume if that works the release scripts wouldn't have a problem

I'd still like to test the windows case however, reached out to their SIG channel to see if there's a pipeline I can run (the check-in tests don't include the publishing info generation steps)

Reply:
sounds good, thanks. just wanted to make sure jq didnt get upset with the removed whitespace.